dev/tabsheet.html

```diff
@@ -28,10 +29,39 @@
         const content = lorem.substring(dotIndex + 1);
         panelContent.textContent = content.repeat(5);
       });
+
+      const tabsheet = document.querySelector('vaadin-tabsheet');
+
+      document.querySelector('#variant-group').addEventListener('value-changed', (e) => {
+        tabsheet.setAttribute('theme', e.detail.value.join(' '));
+      });
+
+      document.querySelector('#direction-group').addEventListener('value-changed', (e) => {
+        document.dir = e.detail.value;
+      });
     </script>
+    <style>
+      vaadin-tabsheet {
+        overflow: hidden;
```

Comment:
Do we actually need this for dev page? It hides the scroller focus-ring which is a bit unexpected.

Reply:
Yes, that makes sense. If you don't mind I would prefer to remove this for now.